package.json

```diff
@@ -28,30 +28,42 @@
   ],
   "scripts": {
     "docs": "jsdoc -c .jsdoc.js",
-    "lint": "eslint '**/*.js'",
-    "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
-    "system-test": "mocha system-test/*.js --timeout 600000",
```

Comment:
I am guessing the timeout is this high for a reason :) Should just keep an eye on the system tests, and see if there's a real need to keep it there

Reply:
I am trying to convert monitoring by following this [PR](https://github.com/googleapis/nodejs-iot/pull/236/files). Looks like that other library use high timeout as well. 
@alexander-fenster @JustinBeckwith  maybe I could try to reduce timeout, but I am sure the meaning. What do you think?

Reply:
This library does not have any system tests other than a packaging test and a trivial smoke test, so there is no need to have high timeout here. Let's just leave `"mocha build/system-test"`.